README.md

```diff
@@ -17,3 +17,83 @@ To publish to the nanopub server you need to setup RSA keys. This allows the nan
 ```
 make_nanopub_keys
 ```
+
+## Quick Start
+
+### Searching for nanopublications
+```python
+from nanopub import NanopubClient
+
+# Create the client, that allows searching, fetching and publishing nanopubs
+npclient = NanopubClient()
+
+# Search for all nanopublications containing the text 'fair'
+results = npclient.search_text('fair')
+print(results)
+
+# Search for nanopublications whose assertions contain triples that are ```rdf:Statement```s.
+# Return only the first three results.
+results = npclient.search_pattern(
+                pred='http://www.w3.org/1999/02/22-rdf-syntax-ns#type',
+                obj='http://www.w3.org/1999/02/22-rdf-syntax-ns#Statement',
+                max_num_results=3)
+print(results)
+
+# Search for nanopublications that introduce a concept that is a ```p-plan:Step```.
+# Return only one result.
+results = npclient.search_things('http://purl.org/net/p-plan#Step', max_num_results=1)
+print(results)
+```
+
+### Fetching nanopublications and inspecting them
+```python
+# Fetch the nanopublication at the specified URI
+np = npclient.fetch('http://purl.org/np/RApJG4fwj0szOMBMiYGmYvd5MCtRle6VbwkMJUb1SxxDM')
+
+# Print the RDF contents of the nanopublication
+print(np)
+
+# Iterate through all triples in the assertion graph
+for s, p, o in np.assertion:
+    print(s, p, o)
+
+# Iterate through the publication info
+for s, p, o in np.pubinfo:
+    print(s, p, o)
+
+# Iterate through the provenance graph
+for s, p, o in np.provenance:
+    print(s,p,o)
+
+# See the list of concepts that are introduced by this nanopublication
+print(np.introduces_concept)
+```
+
+### Publishing an assertion as a nanopub
```

Comment:
IMO publishing is the nicest feature, I would vote for putting this on top?

Reply:
Done!